lib/defaults.js

```diff
@@ -0,0 +1,60 @@
+var defaults = {
+  jquery: {
+    style: 'jquery',
+    noSemver: false,
```

Comment:
The example is a nasty one, since the `[tmp]` syntax is no git standard at all, unlike `!fixup`. I'd rather removing this exception completely.

 But in general it seems easy enough to fix stuff directly in commitplease (who's to stop you?), so yes, let's keep things minimal.